webcompat/views.py

```diff
@@ -133,10 +135,14 @@ def index():
         return render_template('index.html', form=bug_form,
                                browser=browser_name)
     # Form submission.
-    elif request.method == 'POST' and bug_form.validate():
+    elif bug_form.validate_on_submit():
         # copy the form so we can add the full UA string to it.
         form = request.form.copy()
         form['ua_header'] = ua_header
+        # Do we have an image ready to be uploaded?
+        image = request.files['image']
+        if image:
+            form['image_upload'] = json.loads(upload()[0])
```

Comment:
Just noticed that there are flask-uploads… :/

This one
https://github.com/FelixLoether/flask-uploads/
http://flask-uploads.readthedocs.org/en/latest/

and this one
https://pythonhosted.org/Flask-Uploads/
https://bitbucket.org/leafstorm/flask-uploads/
linked from http://flask.pocoo.org/docs/dev/patterns/fileuploads/

There is an issue about that
https://github.com/FelixLoether/flask-uploads/issues/4

Pip seems to install this one.
https://pypi.python.org/pypi/Flask-Uploads/0.1.3
which is the second one. Cool.

The second "official" project has config options for image extensions limiting.
Which there was a size limiting option too.

Though in the doc it says:

>In addition, you can also use patch_request_class to patch your app’s request_class to have a maximum size for uploads. By default, there is no limit, so it’s possible for script kiddies to crash your server by uploading gigantic files. Calling it will install a limit that prevents it from loading more than a certain amount of data.

``` python
patch_request_class(app)        # 16 megabytes
patch_request_class(app, 32 * 1024 * 1024)
                                # 32 megabytes
```